addons/account/models/account_move.py

```diff
@@ -473,6 +473,7 @@ def _sequence_fixed_regex(self):
         string='Total (Tax inc.)',
         help='Use this field to encode the total amount of the invoice.\n'
              'Odoo will automatically create one invoice line with default values to match it.',
+        copy=False,
```

Comment:
Are we sure about this? I would expect it to be copied

Reply:
Quite unsure about this tbh. The idea is here is to prevent a case where:
- the quick encoding is activated
- the user creates an invoice with a quick amount
- the quick encoding is disabled
- the invoice is converted to a misc journal entry (this is the part I'm unsure about)
- the misc journal entry has a quick amount
- the user opens the misc entry, which will call `_get_quick_edit_suggestions` and `self.tax_totals['amount_total]` that will fail

Reply:
That use case doesn't really make sense, and it would already be a corner case if it would make sense

Reply:
Alrighty, pushed without it then